liquibase-core/src/main/java/liquibase/database/core/EnterpriseDBDatabase.java

```diff
@@ -10,7 +10,7 @@
     @Override
     public boolean isCorrectDatabaseImplementation(DatabaseConnection conn) throws DatabaseException {
         final String url = conn.getURL();
-        return url.contains("edb") || (url.contains("postgres") && url.contains(":5444"));
+        return url.contains(":edb:") || (url.contains(":postgres:") && url.contains(":5444"));
```

Comment:
Couldn't the user change the port to something other than 5444? That appears to just be the default.

https://www.enterprisedb.com/edb-docs/d/edb-backup-and-recovery-tool/installation-getting-started/installation-upgrade-guide/2.4/EDB_Postgres_Backup_and_Recovery_Installation_and_Upgrade_Guide.1.15.html

Reply:
The problem it's trying to addess is that EDB users can connect via a jdb:postgres URL and even the connection metadata we get back doesn't let us know it's edb vs. postgresql. This acts as a way to still flag the connection as probably edb in that case. 

If they are using edb and using the jdbc:postgres url and put it on a non-default port, it will fall back to thinking it's postgresql and they'll have to be either OK with that or tell us with the databaseClass argument. I don't think there's any other way we can know the actual db type.

Reply:
Regardless of which path we choose, sounds like we'll have the possibility for false detections on either side. I don't have a suggestion on how to improve this, so I'm not holding up this PR.